.github/pull_request_template.md

```diff
@@ -1,41 +1,7 @@
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] Does the code follow the team's coding standards and best practices?
-- [ ] Is the code well-organized and easy to understand?
-- [ ] Are file, variable and function names clear and meaningful?
 - [ ] Are there any duplicated code blocks that can be refactored into reusable functions?
-
-## Functionality
-- [ ] Does the PR implement the intended functionality or fix the issue described in the title and description?
```

Comment:
Io terrei anche questa riga, almeno come casistica mi capita "spesso" di vedere cose extra dentro una PR che mi portano a farla spezzare in due. Ti riporto l'ultimo caso come esempio

_Per semplificare il lavoro a chi fa le pull request, cerca di tenere separate le modifiche operative (relative alla issue) da quelle che vanno a modificare files che non riguardano la issue per altri motivi (es. csharpier)
Anche per il futuro, se dovessimo andare a vedere cosa è stato modificato per introdurre la funzionalità troveremmo tanto rumore inutile_